src/components/Notifications/TabComponent.tsx

```diff
@@ -4,24 +4,29 @@ import React from 'react';
 interface MyTabComponentProps {
   configuration: React.ReactNode;
   settings: React.ReactNode;
+  activeKey?: number;
 }
 
 export const TabComponent: React.FunctionComponent<
   React.PropsWithChildren<MyTabComponentProps>
-> = (props) => {
-  const [activeTabKey, setActiveTabKey] = React.useState(0);
+> = ({ activeKey = 0, ...props }) => {
+  const [activeTabKey, setActiveTabKey] = React.useState(activeKey);
 
-  const handleTabClick = React.useCallback((tabIndex) => {
-    setActiveTabKey(tabIndex);
-  }, []);
+  const handleTabSelect = (
+    // eslint-disable-next-line
+    event: React.MouseEvent<HTMLElement, MouseEvent>,
```

Comment:
No need to disable eslint. It should be enough to consume the unused event as `_e`